SofaKernel/framework/sofa/helper/system/PluginManager.cpp

```diff
@@ -87,12 +87,18 @@ PluginManager::~PluginManager()
 void PluginManager::readFromIniFile(const std::string& path)
 {
     std::ifstream instream(path.c_str());
-    std::string pluginPath;
-
-    while(std::getline(instream,pluginPath))
+    std::string pluginPath, line, version;
+    while(std::getline(instream, line))
     {
+        if (line.empty()) continue;
+
+        std::istringstream is(line);
+        is >> pluginPath;
+        is >> version; // information not used for now
```

Comment:
The old ini file does not have the version number. 
Maybe you can implement a backward compatibility hook, I'm not sure this could work like that but 
here is the idea:
```cpp
   is >> pluginPath
   if( is.eof() ){
      msg_deprecated() << "You are using a deprecated version of the fiel.ini... blabblha"
   ...
```
Why not having the version number into the filename (as linux is doing libpng.1.0.so) ?